test/Microsoft.AspNet.Mvc.ModelBinding.Test/Binders/KeyValuePairModelBinderTest.cs

```diff
@@ -28,8 +26,7 @@ public class KeyValuePairModelBinderTest
             var retVal = await binder.BindModelAsync(bindingContext);
 
             // Assert
-            Assert.NotNull(retVal);
-            Assert.Null(retVal.Model);
+            Assert.Null(retVal);
```

Comment:
the only non-comment change I made to `KeyValuePairModelBinder` was to remove extra special case returns, leaving that to the existing handling in `GenericModelBinder`.

so, this isn't a `null` with errors that will ever be visible.  just looks `null` temporarily because we're testing `KeyValuePairModelBinder` instead of its `GenericModelBinder` caller.  `GenericModelBinder` never returns `null` if it can find a binder.